script/8_deploySwapAndBridge.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+echo "Instructing the shell to exit immediately if any command returns a non-zero exit status..."
+set -e
+echo "Done."
+
+echo "Navigating to the root directory of the project..."
+cd ../
+echo "Done."
+
+echo "Setting environment variables..."
+source .env
+echo "Done."
+
+echo "Creating $NETWORK directory inside deployment directory..."
+if [ -z "$NETWORK" ]
+then
+      echo "NETWORK variable inside .env file is not set. Please set NETWORK environment variable."
+      exit 1
+else
+      if [ -d "deployment/swap_and_bridge/$NETWORK" ]
+      then
+            echo "Directory deployment/swap_and_bridge/$NETWORK already exists."
+      else
+            mkdir -p deployment/swap_and_bridge/$NETWORK
+      fi
+fi
+echo "Done."
```

Comment:
Is this actually needed?
Does any of below scripts read/write something inside `deployment` directory?
Otherwise it may be removed.

Reply:
removed